.github/dependabot.yml

```diff
@@ -14,6 +14,9 @@ updates:
       interval: "weekly"
     rebase-strategy: "disabled"
     open-pull-requests-limit: 10
+    ignore:
+      - dependency-name: python
+        versions: [">=3.12"]
```

Comment:
I never know what to do in situations like this where there are multiple places in the code base related to a single thing, but they're not connected.

For example, we have this ignore bit here and it's tied to the docker base image we use. When we update the docker base image, we need to remember to update this, too. Will we? What helps us remember? Should we add a comment to `docker/Dockerfile`? Should we have a runbook for updating the Python version?

Reply:
The Python version is indeed used in many places. Having a list of all these places in Dockerfile seems useful. I'll add it.